evm-tests/jsontests/tests/state.rs

```diff
@@ -54,9 +80,13 @@ const SKIPPED_CASES: &[&str] = &[
 	"stTransactionTest/HighGasPrice",
 	"stTransactionTest/HighGasPriceParis",
 	"stCreateTest/CreateTransactionHighNonce",
+	// KZG-precompile not supported
```

Comment:
I assume this is coming as part of a future PR.

Reply:
Yes, it will be separated PR related to `EIP-4844` tests.